packages/example-codehub/src/codehub-application.ts

```diff
@@ -25,13 +28,13 @@ export class CodeHubApplication extends Application {
     app.controller(UserController);
     app.controller(HealthController);
 
-    app.bind('userId').to(42);
+    server.bind('userId').to(42);
 
-    app.bind('app.info').toDynamicValue(() => this.info());
+    server.bind('app.info').toDynamicValue(() => this.info());
 
-    app.bind('servers.http.enabled').to(true);
-    app.bind('servers.http.port').to(3000);
-    app.bind('servers.https.enabled').to(true);
+    server.bind('http.enabled').to(true);
+    server.bind('http.port').to(3000);
+    server.bind('https.enabled').to(true);
```

Comment:
I think these three bindings can be actually removed, they are not supported by the current implementation. Maybe with the exception of `http.port`, which should use `RestBindings.HTTP_PORT` instead?